cmd/gator/verify/verify.go

```diff
@@ -89,10 +89,11 @@ func runE(cmd *cobra.Command, args []string) error {
 	if strings.HasSuffix(originalPath, "/...") {
 		recursive = true
 		targetPath = strings.TrimSuffix(targetPath, "...")
+		originalPath = strings.TrimSuffix(originalPath, "...")
```

Comment:
in this file, right above lines 68-79 see how we create a `fileSystem` .

Since we pass that in and the absolute path, it's hard to print the results with the input path, given that we can traverse a directory.

Another way to tackle this problem, where we don't pass in the `InputPath` would be to push down the `fileSystem`/ absolute path creation to the point where we walk the directory or where we try to read a file. But I think that approach would require a more sweeping refactor.